cmd/goose/cmd_dbversion.go

```diff
@@ -9,9 +9,10 @@ import (
 
 var dbVersionCmd = &Command{
 	Name:    "dbversion",
-	Usage:   "",
+	Usage:   "dbversion",
 	Summary: "Print the current version of the database",
-	Help:    `dbversion extended help here...`,
+	Help:    `dbversion introspects the database and prints out the last migration
```

Comment:
Perhaps something like `dbVersion prints out the last migration applied` might be a little more clear.